pylintrc

```diff
@@ -0,0 +1,561 @@
+[MASTER]
```

Comment:
Pylint is a static analysis tool that helps to make sure we're writing good code. It also gets run in Travis like flake8.